### PR TITLE
1845 live viewer main window

### LINKED ISSUE #1845: Live Viewer: Develop Front-end window
New live viewer window

Should be shown from `MainWindowView.show_live_viewer()`, which is in #1849

### PR DESCRIPTION
**Pull request by JackEAllen**
### Issue

Closes #1845 

### Description

Create basic Live Viewer Window Displaying the first .tif or .tiff image file read from a selected dataset

### Acceptance Criteria 

User can select a directory containing .tif / .tiff files from Mantid Imaging `File>Live Viewer` and a new live viewer window opens only on selecting a dataset  displaying one image from the selected directory.

### Documentation

See Next Release Notes

### CODE REVIEW

File: mantidimaging/gui/windows/live_viewer/view.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    image_load_dialog: Optional[ImageLoadDialog] = None
+
+    tableView: RemovableRowTableView
+
+    normaliseCheckBox: QCheckBox
+    imageLayout: QVBoxLayout
+    exportButton: QPushButton
+    normaliseErrorIcon: QLabel
+    _current_dataset_id: Optional['UUID']
+    normalise_error_issue: str = ""
```
Comment (samtygier-stfc) [inline mantidimaging/gui/windows/live_viewer/view.py:40]: Most of these can go. I think only `imageLayout` that we want to keep.

File: mantidimaging/gui/windows/live_viewer/view.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    def __init__(self, main_window: 'MainWindowView', live_dir_path: Path) -> None:
+        super().__init__(main_window, 'gui/ui/live_viewer_window.ui')
+        self.setWindowTitle("Mantid Imaging - Live Viewer")
+        self.main_window = main_window
+        self._path = live_dir_path
+
+        self.presenter = LiveViewerWindowPresenter(self, main_window)
+
+        self.main_window = main_window
```
Comment (samtygier-stfc) [inline mantidimaging/gui/windows/live_viewer/view.py:50]: Duplication

File: mantidimaging/gui/windows/live_viewer/view.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+        icon_path = finder.ROOT_PATH + "/gui/ui/images/exclamation-triangle-red.png"
+        self.normalise_error_icon_pixmap = QPixmap(icon_path)
```
Comment (samtygier-stfc) [inline mantidimaging/gui/windows/live_viewer/view.py:53]: Can remove these

File: mantidimaging/gui/windows/live_viewer/view.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    @property
+    def path(self):
+        """Path to the directory containing the images"""
+        return self._path
+
+    @path.setter
+    def path(self, path: Path):
+        """Set the path to the directory containing the images to watch"""
+        self._path = path
```
Comment (samtygier-stfc) [inline mantidimaging/gui/windows/live_viewer/view.py:68]: As we are not doing anything fancy with `path` it can just be a regular public attribute.

**Review by samtygier-stfc — CHANGES_REQUESTED** [review]
A few left over bits.

**Comment by JackEAllen** [comment]
> A few left over bits.

Thanks for noticing these 👍

**Review by samtygier-stfc — APPROVED** [review]
Looks good

**Pull request merged**